tests/epyccel/test_generic_functions.py

```diff
@@ -0,0 +1,94 @@
+ # pylint: disable=missing-function-docstring, missing-module-docstring/
+import pytest
+import modules.generic_functions as mod
+from pyccel.epyccel import epyccel
+
+@pytest.fixture( params=[
+        pytest.param("fortran", marks = [
+            pytest.mark.fortran,
+            pytest.mark.skip]),
+        pytest.param("c", marks = [
+            pytest.mark.c]
```

Comment:
Does this PR already implement issue #512 ?

Reply:
No it doesn't, I am pyccelizing the modules in the tests

Reply:
Ah ok. So you don't actually call any interface functions directly here :+1: